### ISSUE TITLE
ARM Support for CPU mining

### ISSUE THREAD

**Issue by Cryptominer937** [RainbowMiner/RainbowMiner#1090]
I'm currently working on building as many miners as i can for Pi 4 armV7l with and without neon support. 

My question is this

If I manage to build them and attach/link them here would you be willing to put the time in to make RBM work on Pi 4?

**Comment by RainbowMiner** [comment]
Yes, of course. Which Pi4 model will you be using, exactly? I consider buying one, so that I can try out myself.

**Comment by Cryptominer937** [comment]
I have the Raspberry PI 4 2GB
Currently it's running Raspbian 32 bit
I will experiment with 32 vs 64 bit speeds

**Comment by RainbowMiner** [comment]
I see - I think, 2GB is pretty tight, so I'll get me a 4GB or 8GB version. Then I'll add ARM installation to RainbowMiner and shrink the miner selection to ARM compatible miners, to reduce the calculation load.

**Comment by Cryptominer937** [comment]
Yea, I've had to compile all the ones I have working. Only 1 I had to adjust code on to get working so far.
